.github/fabricbot.json

```diff
@@ -1,1470 +0,0 @@
-[
-  {
-    "taskSource": "fabricbot-config",
-    "taskType": "trigger",
-    "capabilityId": "IssueResponder",
-    "subCapability": "IssuesOnlyResponder",
-    "version": "1.0",
-    "config": {
-      "taskName": "Add untriaged label to new/reopened issues without a milestone",
-      "actions": [
-        {
-          "name": "addLabel",
-          "parameters": {
-            "label": "untriaged"
-          }
-        }
-      ],
-      "eventType": "issue",
-      "eventNames": [
-        "issues"
-      ],
-      "conditions": {
-        "operator": "and",
-        "operands": [
-          {
-            "operator": "or",
-            "operands": [
-              {
-                "name": "isAction",
-                "parameters": {
-                  "action": "opened"
-                }
-              },
-              {
-                "name": "isAction",
-                "parameters": {
-                  "action": "reopened"
-                }
-              },
-              {
-                "name": "removedFromMilestone",
-                "parameters": {}
-              }
-            ]
-          },
-          {
-            "name": "isOpen",
-            "parameters": {}
-          },
-          {
-            "operator": "not",
-            "operands": [
-              {
-                "name": "isInMilestone",
-                "parameters": {}
-              }
-            ]
-          },
-          {
-            "operator": "not",
-            "operands": [
-              {
-                "name": "hasLabel",
-                "parameters": {
-                  "label": "untriaged"
-                }
-              }
-            ]
-          }
-        ]
-      }
-    }
-  },
-  {
-    "taskSource": "fabricbot-config",
-    "taskType": "trigger",
-    "capabilityId": "IssueResponder",
-    "subCapability": "IssuesOnlyResponder",
-    "version": "1.0",
-    "config": {
-      "taskName": "Remove untriaged label from issues when closed or added to a milestone",
```

Comment:
This task did not get converted, presumably because it uses the `addedToMilestone` action which is not supported.